src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -206,6 +219,12 @@ export default class PathwayMapperTable extends React.Component<
 
     render() {
         const columns = _.sortBy(this._columns);
+        console.log(this.props);
+        /*this.dataStore.page = findPageOfSelectedPathway(
+            this.props.data,
+            this.dataStore,
+            this.props.selectedPathway
+        );*/
```

Comment:
also remove this one for now. we can address this page issue later in a separate PR.